<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SnakeGame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>875</y>
      <width>200</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/button.png);</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>665</y>
      <width>200</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/button.png);</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>770</y>
      <width>200</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/button.png);</string>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelScoreText">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>365</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前分：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelScore">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>365</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHighScoreText">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>260</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>最高分：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHighScore">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>260</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="bulletStatusLabel">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>555</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>子弹：就绪</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInvincibleRemainingText">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>450</y>
      <width>150</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>无敌剩余时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInvincibleRemaining">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>450</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
